### PR TITLE
🇩🇰EID-1550: Add in new pipeline to deploy Denmark integration

### PR DESCRIPTION
**Pull request by tommrr**

### CODE REVIEW

**Comment by blairboy362** [comment]
Before this will work you need to add an istio `DestinationRule` and `VirtualService` (follow the `hub-egress-` examples as a template. You'll also need to add an entry for `eidasconnector.test.eid.digst.dk` to `verify-cluster-config`.

Nitpick: the changes look more like a "deploy" than a "build" (title of the PR had me confused for a moment).

**Review by instantiator — DISMISSED** [review]

**Comment by CharlesIC** [comment]
This is good to go. I can't approve because I've pushed to this PR.

**Review by blairboy362 — DISMISSED** [review]

**Comment by blairboy362** [comment]
@CharlesIC I think you can approve it...I think it only doesn't count for the person that opened (owns) the pull request. I'm sure I've approved PRs I've pushed to in the past...

File: ci/integration/deploy-pipeline.yaml [hunk 1]
```diff
@@ -51,6 +51,103 @@ spec:
...
+          params:
+            CLUSTER_NAME: ((cluster.name))
+            CLUSTER_DOMAIN: ((cluster.domain))
+            RELEASE_NAME: dk-integration
+            CONNECTOR_NODE_NATIONALITY_CODE: DK
+            RELEASE_NAMESPACE: ((namespace-deployer.namespace))
+            CLUSTER_PRIVATE_KEY: ((cluster.privateKey))
+            ERROR_PAGE_URL: https://www.integration.signin.service.gov.uk/proxy-node-error
+            CONNECTOR_ENTITY_ID: https://eidasconnector.test.eid.digst.dk/Metadata
+            CONNECTOR_METADATA_FQDN: eidasconnector.test.eid.digst.dk
```
Comment (idrop) [inline ci/integration/deploy-pipeline.yaml]:
```suggestion
            CONNECTOR_METADATA_FQDN: eidasconnector.test.eid.digst.dk
            HUB_FQDN: www.integration.signin.service.gov.uk
```
I think the `HUB_FQDN` was added to nl-integration recently, and we should add this too

**Review by idrop — CHANGES_REQUESTED** [review]

**Review by idrop — DISMISSED** [review]
Good to go

**Review by instantiator — APPROVED** [review]

**Review by blairboy362 — APPROVED** [review]

**Pull request merged**